braindecode/augmentation/functional.py

```diff
@@ -144,6 +150,10 @@ def _ft_surrogate(x=None, f=None, eps=1, random_state=None):
     eps: float, optional
         Float between 0 and 1 setting the range over which the phase
         pertubation is uniformly sampled: [0, `eps` * 2 * `pi`]. Defaults to 1.
+    channel_indep : bool, optional
+        Whether to sample phase perturbations independently for each channel or
+        not. It is advised to set it to False when spatial information is
+        important for the task, like in BCI. Default False.
     random_state: int | numpy.random.Generator, optional
         By default None.
```

Comment:
One question, why do we even need comment for private function, especially if public function has same comment again? Can't we just remove it here to avoid duplicate docs? Also do we need default argument for channel_indep here? like a bit confusing for me there is a default here but not at the public one

Reply:
I don't see a problem in the comment in a private function. This is useful for the developer who proposes a new augmentation or even to explain the library logic to a student. 

In that case, I prefer to err on the side of excess. Also, some augmentations take inspiration from other libraries.

But I don't know if there is any rule about it.

About the default value, I'm not sure.

Reply:
I am not sure about the additional comment, it is more a matter of taste to me.

About the default value: I would say that either both functions should have a default value or neither of them. As a user typically only uses the `Transforms` anyway, I don't think that this decision matters too much.

To me, both "problems" raise the same question: why do we even need a private function?

Reply:
I think that in any case duplication should be avoided where possible, especially comment duplication as duplicate comments require extra work to keep in sync and can easily go out of sync as well

Reply:
What do you think @cedricrommel?

Reply:
TBH I agree with @robintibor and @martinwimpff although I implemented the private function :) At some point it made sense because I allowed to isolate the official code from the authors, but it makes no sense anymore, so we should not have the private version anymore. I'll fix this ASAP.